File: Programming Basics with Python/T04 - Conditional Statements (Advanced)/exercise/05. Journey.py
budget = float(input())
season = input()
destination = ""
type_holiday = ""
money_spent = 0

if budget <= 100:
    destination = "Bulgaria"
    if season == "summer":
        type_holiday = "Camp"
        money_spent = budget * 0.30
    elif season == "winter":
        type_holiday = "Hotel"
        money_spent = budget * 0.70

elif budget <= 1000:
    destination = "Balkans"
    if season == "summer":
        type_holiday = "Camp"
        money_spent = budget * 0.40
    elif season == "winter":
        type_holiday = "Hotel"
        money_spent = budget * 0.80

elif budget > 1000:
    destination = "Europe"
    type_holiday = "Hotel"
    money_spent = budget * 0.90

print(f"Somewhere in {destination}")
print(f"{type_holiday} - {money_spent:.2f}")

File: Programming Basics with Go/T04 - Conditional Statements (Advanced)/exercise/05. Journey.go
package main

import "fmt"

func main() {
	var budget float64
	fmt.Scanln(&budget)

	var season string
	fmt.Scanln(&season)

	var destination string = ""
	var price float64 = 0
	var place string = ""

	if budget <= 100 {
		destination = "Bulgaria"
		switch season {
		case "summer":
			price = budget * 0.3
			place = "Camp"
			break
		case "winter":
			price = budget * 0.7
			place = "Hotel"
			break
		}
	} else if budget <= 1000 {
		destination = "Balkans"
		switch season {
		case "summer":
			price = budget * 0.4
			place = "Camp"
			break
		case "winter":
			price = budget * 0.8
			place = "Hotel"
			break
		}
	} else if budget > 1000 {
		destination = "Europe"
		price = budget * 0.9
		place = "Hotel"
	}
	fmt.Printf("Somewhere in %s\n", destination)
	fmt.Printf("%s - %.2f", place, price)
}
